### ISSUE TITLE
Wire can’t access your system's safe storage.

### ISSUE THREAD

**Issue by KitchM** [wireapp/wire-desktop#7890]
Wire version: Version 3.35.3348
Wire for web version: 
Operating system:  Debian 11 Bullseye, Xfce 4.16
Which antivirus software do you have installed: 

What steps will reproduce the problem?

1.  ...Start Wire Desktop
2. Attempt to log in.
3.  ...
4.  ...

What is the expected result?

...

What is the actual result?

...Wire can’t access your system's safe storage.

Please provide any additional information below. Attach a screenshot if possible.
This issue appears to be a problem with the internal encryption key storage in the OS.  If that is so, then Wire does not automatically utilize the existing system.  Since that mechanism of security keys can be handle in any of various manners, then why is it not standardized as other programs address it?  If a particular system needs to be installed and configured a particular way, then it appears that Wire is not as utilitarian as needed.

...
![Screenshot_2024-05-27_11-35-39 - Wire Desktop](https://github.com/wireapp/wire-desktop/assets/371308/6ab94268-f8fc-4862-aa51-a0cff67f10dd)

**Comment by kevlu93** [comment]
Have this issue as well on Arch Linux (6.9.1-arch1-2 kernel version)

**Comment by KitchM** [comment]
Yes, I'm sure that it isn't just me, but rather a fundamental failure to check for dependencies, and maybe alternatives to some common methods for handling keys and such.

**Comment by KjeldJensen** [comment]
I have the same issue on Ubuntu 24.04 
Tested using Wire 3.32.3079 and Wire 3.35.3348 (available in the Ubuntu App center)

**Comment by KitchM** [comment]
Well, mine just did something different.  It auto-started but this time shows me as logged in already.  Do I selected Conversation>Start, and then it came up that I had to log in after all.

It is set to auto-start every day.  Nothing in my system has changed since yesterday.  There is definitely inconsistent behavior in the program.

**Comment by dm17** [comment]
Same using the flatpak.

**Comment by KitchM** [comment]
Well, this is interesting.  I just followed the instructions found at [https://github.com/wireapp/wire-desktop/wiki/How-to-install-Wire-for-Desktop-on-Linux](url) on my new Debian 12.  The Wire Desktop installed correctly and I was able to log in without a problem.

It is interesting because I just removed GNOME Keyring and rebooted using KeePassXC with the SecretService.  Hmm, I wonder........
